### ISSUE TITLE
Can't install the package mplex

### ISSUE THREAD

**Issue by svish91** [Achab94/mplex#4]
Hi,

I am trying to install the package but can't install it. R is showing me this error:
Downloading GitHub repo Achab94/mplex@master
Error in (function (command = NULL, args = character(), error_on_status = TRUE,  : 
  System command error

Please fix this bug.

**Comment by Achab94** [comment]
Strange, have you tried with these two commands:
```
install.packages("devtools")
devtools::install_github("achab94/mplex")
```
Let me know

**Comment by svish91** [comment]
Yes I tried that only. When I type the second command, I get the error.

**Comment by svish91** [comment]
I tried just now as well:
I got this error
  (converted from warning) installation of package '/var/folders/z9/sv_xq7693tn3c60z8rc6cx3r0000gn/T//RtmpSFKPk6/file2e97796bbeb/mplex_1.0.tar.gz' had non-zero exit status

**Comment by Achab94** [comment]
Actually I'm trying it and there seems to be some troubles with devtools. Are you able to `library(devtools)` without any warnings/errors?

**Comment by svish91** [comment]
I fixed the issue. You are right devtools have some issues for dependency rcmdcheck.
Your function works. Thank a lot for your support.

**Comment by Achab94** [comment]
No problem, glad to help you :) My package needs lot of improvements actually, one day I'll have time to do it. I think there are also libraries on Python for managing multiplex networks which are for sure developed better than mine :)

**Comment by svish91** [comment]
Thanks for your help and suggestions!
